contrib/githooks/README.md

```diff
@@ -0,0 +1,7 @@
+# Git hooks
```

Comment:
Maybe add a short description of what this does under the hood? i.e. `gofmt`, etc.